.devcontainer/codespaces-docs/welcome.rst

```diff
@@ -1,5 +1,60 @@
-Welcome to GitHub PyMAPDL Codespaces for Documentalist!!
-========================================================
 
-This codespace has been specifically set for documenting PyMAPDL.
+===========================================================
+Welcome to GitHub PyMAPDL Codespaces for Documentalist!! 📚
+===========================================================
```

Comment:
```suggestion
=========================================================
Welcome to GitHub PyMAPDL Codespaces for Documentalist 📚
=========================================================
```
Avoid exclamation points in technical doc. Is "Documentalist" correct? I'm not familiar with this term (yet).

Reply:
I don't know... whe I wrote "documentalist" I was thinking in people like you.. who works with docs constantly.. they dont need to test the library.. but build their docs.

Reply:
the `devcontainer-local` and the one at the root (`codespaces-dev`) is for developer... no docs dependencies are installed (although you could).